app/src/main/res/layout/user_card_list.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:cardBackgroundColor="@color/cardview_light_background"
+    android:clickable="true"
+    android:focusable="true"
+    android:layout_marginLeft="16dp"
+    android:layout_marginRight="16dp"
+    app:cardCornerRadius="20dp"
+    android:layout_marginTop="8dp">
+
+<android.support.constraint.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingStart="10dp"
+    android:paddingTop="15dp"
+    android:paddingBottom="15dp">
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/imageView"
+        android:src="@drawable/ship_rain"
+        app:civ_border_width="2dp"
+        app:civ_border_color="#FF000000"
+        android:layout_width="107dp"
+        android:layout_height="107dp"
+        android:layout_marginEnd="16dp"
+        android:contentDescription="@string/user_image"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="1.0"
+        android:elevation="10dp"
+        />
+
+    <TextView
+        android:id="@+id/textView2"
```

Comment:
@omokolataiwo
name your ids according to best practices
https://github.com/futurice/android-best-practices#resources